Day-6-7/src/index.css

```diff
@@ -74,3 +74,76 @@ body {
   text-align: center;
   margin-bottom: 25px;
 }
+
+.user-container {
+  color: #fff;
+  border: 1px solid;
+  border-radius: 5px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  margin: 150px;
+  min-height: 700px;
+  /* max-width: 1000px; */
+  position: relative;
+  padding: 40px;
+}
+
+.user-img {
+  width: 150px;
+  height: 150px;
+  border-radius: 50%;
+  position: absolute;
+  top: -50;
+}
+
+.user-title {
+  margin-top: 50px;
+  border-bottom: 1px solid lightgray;
+  margin-bottom: 10px;
+}
+
+.user-bio {
+  color: lightgray;
+  font-size: 18px;
+}
+
+.repo-title {
+  margin-top: 15px;
+  color: #fed669;
+}
+
+.repo-container {
```

Comment:
can use underscores while naming classes. We have a benefit that if they are too long then you can press ``command D + command c`` on VS CODE and it copies taking all of the class. Using a hyphen "-", you have to do all of the copying in the hard way.

Reply:
ok sure!